src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj

```diff
@@ -16,6 +16,7 @@
 		- Added support for SecureInput in activities
 		- Added ScriptActions to on demand HDI linked service
 		- Added support for recursive Cancel operation on runs
+		- Added TumblingWindowRerunTrigger API
```

Comment:
version `2.3.0` is already published, is this a part of a newer release?

Reply:
Yes, in 2 weeks @hvermis will update the version when the team is ready to release